distribution/manifest.go

```diff
@@ -197,19 +196,12 @@ func detectManifestBlobMediaType(dt []byte) (string, error) {
 			return "", fmt.Errorf(`media-type: %q should not have "config", "layers", or "fsLayers"`, mfst.MediaType)
 		}
 		return mfst.MediaType, nil
-	case schema1.MediaTypeManifest:
```

Comment:
Can we return a human-friendly error on this `case`?

Reply:
Ah, yes, I guess for these I tried to keep the deprecated packages out; let me have another look at that (and if needed, just copy the `schema1.MediaTypeManifest` value